cmake/DD4hepBuild.cmake

```diff
@@ -779,7 +779,7 @@ MACRO(DD4HEP_SETUP_BOOST_TARGETS)
   FOREACH(FS_LIB_NAME stdc++fs "" c++fs )
     dd4hep_debug("|++++> linking against ${FS_LIB_NAME}")
     try_compile(HAVE_FILESYSTEM ${CMAKE_BINARY_DIR}/try ${DD4hep_DIR}/cmake/TryFileSystem.cpp
-      CXX_STANDARD ${CMAKE_CXX_STANDARD}
+      CXX_STANDARD "${CMAKE_CXX_STANDARD}"
```

Comment:
Please remove this change, in case we ever re-arrange the dd4hep cmake we should error out if we don't set the standard before calling this.